docs/core/epochTime.md

```diff
@@ -0,0 +1,11 @@
+# Epoch time
+
```

Comment:
DDK timeservice uses UNIX format Epoch time for determining time of the blocks and other network events.